README.md

```diff
@@ -16,7 +16,34 @@ http://developers.google.com/api-client-library/php
 
 ## Installation ##
 
-For the latest installation and setup instructions, see [the documentation](https://developers.google.com/api-client-library/php/start/installation).
+The [Releases](https://github.com/google/google-api-php-client/releases) page lists all stable versions. Download any file
+with the name `google-api-php-client-[RELEASE_NAME].zip` for a package including this library and its dependencies.
+
+### Composer
+
+Install this library via [composer](https://getcomposer.org) by running the following command:
+
+```sh
+# change directories to the root of your project
+cd /path/to/my-project
+# if you don't already have composer, install it
+curl -sS https://getcomposer.org/installer | php
+# install this library
+./composer.phar require google/apiclient
+```
+
+> The above example adds a file `composer.phar` to your project working directory for simplicity. We recommend
+> you [install composer globally](https://getcomposer.org/doc/00-intro.md#globally) for local development.
+
+### Autoloading
+
+Once installed, be sure to include the autoloader:
+
+```php
+require_once '/path/to/google-api-php-client/vendor/autoload.php';
```

Comment:
yeah, hmm... I mean't it specifically for those who downloaded the release w/o composer. But you're right, it's confusing.